modules/mobile/docs/openapi.yaml

```diff
@@ -1122,6 +1122,205 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Errors"
+  /v0/push/register:
+    post:
+      summary: /v0/push/register
+      description: Allows a new app install to register to receive push notifications
+      operationId: register
+      security:
+        - Bearer: [ ]
+      requestBody:
+        description: Device information
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PushRegistration'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PushRegistration'
```

Comment:
What is an appropriate response for a basic endpoint like this? The same data that was passed, or just 200 OK? What is the norm with the rest of the mobile api?

Reply:
for updates (PUT) we use just a 200 with no body, since this is a POST could it be a 201 created (also no body)?